### PR TITLE
manatee is fucking cringe as shit: mimes can now get the mech that was designed for them

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
Pretty cool item for a role that unfortunately has next to no exclusive items, which is pretty sad. The things the mech comes equipped with aren't terribly overpowered, so this is probably balanced for 20tc. 

# Changelog

:cl:  
rscadd: traitor mimes can buy the mime mech for 20tc
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
Gus redemption arc?

**Comment by Moltijoe** [comment]
i had no clue the reticence was even in the codebase, why wasn't it available before?

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> why wasn't it available before?

idk dawg i aint a coder

**Comment by Moltijoe** [comment]
> idk dawg i aint a coder

says the guy that just did a coding

**Comment by ChuChu42** [comment]
it is fair since clown gets a shit ton of cool gear
plus mime mech was unused

also what weapons does the mime mech have?

**Comment by Mqiib** [comment]
I think there's a loaded subtype of mechs that start with equipment. Might want to see if that exists and use it or, if it doesn't, make it. That is, assuming it's supposed to come with the equipment the wiki says it does.

**Comment by ChuChu42** [comment]
i have several issues from testing the mech
1. power issues, too low power, silenced gun and RCD both use power, along with moving, runs out too fast
2. the "silenced" pistol makes a syringe gun noise therefore its not even silent
3. 100 integrity low armor, we can say this is a glass cannon but come on its 20 tc it needs more.
4. punches and RCD are not silent either


if we wanna keep this as a glass cannon please at least make everything actually silenced instead of saying "silenced" and being able to be heard thru walls

**Comment by ChuChu42** [comment]
> I think there's a loaded subtype of mechs that start with equipment. Might want to see if that exists and use it or, if it doesn't, make it. That is, assuming it's supposed to come with the equipment the wiki says it does.

it DOES have a loaded subtype which is not the one that spawns at CC

**Review by MajManatee — CHANGES_REQUESTED** [review]
make the above mentioned equipment actually silent and sure

**Comment by ChuChu42** [comment]
> make the above mentioned equipment actually silent and sure

you forgot to use the loaded subtype, this spawns a mech with no equipment.

**Review by Moltijoe — APPROVED** [review]

**Comment by ChuChu42** [comment]
from testing: due to how lights work

the mech has a natural green glow, which makes it easily be able to seen, whould be appriciated if you can try removing that glow

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2265,6 +2265,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	manufacturer = /datum/corporation/traitor/waffleco
 	surplus = 0
 
+/datum/uplink_item/role_restricted/reticence
+	name = "Retience Exosuit"
+	desc = "A popular product for pantomime performers. For when you need to silence a target a little more... permanently. Comes equipped with advanced cloaking technology, a Quietus carbine, and an RCD."
+	item = /obj/mecha/combat/reticence/loaded
```
Comment (ReddicusDragon) [inline code/modules/uplink/uplink_items.dm:2271]: does it actually have the weapons?

**Review by ReddicusDragon — APPROVED** [review]

**Review by ReddicusDragon — APPROVED** [review]

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
I am fucking stupid this still needs to be silent, the punch needs to be silent, the move needs to be silent, and the RCD needs to be silent. According to manatee.

**Comment by ChuChu42** [comment]
> I am fucking stupid this still needs to be silent, the punch needs to be silent, the move needs to be silent, and the RCD needs to be silent. According to manatee.

and the green light needs to be removed for actual invisiblity.

**Comment by TheRyeGuyWhoWillNowDie** [comment]
eh

**Comment by ChuChu42** [comment]
> eh

why

**Pull request closed without merging**